flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/utils/RelExplainUtil.scala

```diff
@@ -858,4 +859,28 @@ object RelExplainUtil {
       case (k, v) => s"$k = (${v.mkString(", ")})"
     }.mkString(", ")
 
+  /**
+   * Converts [[RelHint]]s to String.
+   */
+  def hintsToString(hints: util.List[RelHint]): String = {
```

Comment:
Why not simply `hints.toString()`? I just find the result is a little different with `hints.toString()` because this doesn't add `,` between hint elements.

Reply:
The difference between the result of this method and `hints.toString()` is we ignore the `inheritPath` if it's empty in this method. Because most case, the inheritPath is empty, and the result of this method is much simpler.